app/controllers/file_attachments_controller.rb

```diff
@@ -58,6 +58,13 @@ def edit
   end
 
   def update
-    redirect_to file_attachments_path
+    result = FileAttachments::UpdateInteractor.call(params: params,
+                                                   user: current_user)
+    edition, attachment_revision = result.to_h
+                                         .values_at(:edition,
+                                                    :file_attachment_revision)
```

Comment:
Indentation here seems slightly odd to me, how about this? 
```ruby
result.to_h.values_at(:edition,
                      :file_attachment_revision)
```

Reply:
I was trying to avoid going too far over the max character count for the line.  That implementation goes too far 😞

Reply:
If it helps, we do [break with this convention sometimes](https://github.com/alphagov/content-publisher/blob/master/app/controllers/images_controller.rb#L55) when it's not helpful :-).

Reply:
+1 to Ben's suggestion.

Reply:
Updated as per Ben's suggestion 👍